test/klusterlet/klusterlet.yaml

```diff
@@ -18,5 +18,5 @@ spec:
     featureGates:
       - feature: AddonManagement
         mode: Enable
-  registrationImagePullSpec: 'quay.io/open-cluster-management/registration:v0.8.0'
-  workImagePullSpec: 'quay.io/open-cluster-management/work:v0.8.0'
+  registrationImagePullSpec: 'quay.io/open-cluster-management/registration:v0.9.0'
+  workImagePullSpec: 'quay.io/open-cluster-management/work:v0.9.0'
```

Comment:
I think can be dropped and we can use the alm-example from the csv (see previous commit).

Reply:
In both cases, yes it can be used instead of adding another yaml to our repo.